x/records/keeper/lsm_token_deposit_test.go

```diff
@@ -93,28 +93,28 @@ func (s *KeeperTestSuite) TestAddLSMTokenDeposit() {
 	// verify that for previously non-existing deposits the amounts set
 	for i := 5; i < 10; i++ {
 		new := newDeposits[i]
-		actual, found := s.App.StakeibcKeeper.GetLSMTokenDeposit(s.Ctx, new.ChainId, new.Denom)
+		actual, found := s.App.RecordsKeeper.GetLSMTokenDeposit(s.Ctx, new.ChainId, new.Denom)
 		s.Require().True(found, "deposit not found in store %+v", new)
 		s.Require().Equal(new.Amount, actual.Amount,
 			"found amount %d did not match expected amount %d ", actual.Amount, new.Amount)
 	}
 }
 
 func (s *KeeperTestSuite) TestUpdateLSMTokenDepositStatus() {
-	statuses := []types.LSMDepositStatus{
-		types.TRANSFER_IN_PROGRESS,
-		types.TRANSFER_FAILED,
-		types.DETOKENIZATION_QUEUE,
-		types.DETOKENIZATION_IN_PROGRESS,
-		types.DETOKENIZATION_FAILED,
+	statuses := []types.LSMTokenDeposit_Status{
+		types.LSMTokenDeposit_TRANSFER_IN_PROGRESS,
```

Comment:
We should add one for the DEPOSIT_QUEUE and update the expected test behavior but maybe it should its own small PR.  That state didn't exist when these tests were written and accordingly I don't think any tests test for the transitions between DEPOSIT_QUEUE and TRANSFER_IN_PROGRESS

Reply:
assuming you mean DEPOSIT_PENDING, yeah I think it'd be good to update those - wanna take a stab at that today?